roles/jupyterhub/tasks/packages.yml

```diff
@@ -1,14 +1,14 @@
 ---
 
 - name: JupyterHub conda dependencies
-  conda: name={{item}} state=present
+  command: conda install name={{item}} state=present
```

Comment:
Thanks Min. These are module names not just aliases to the commands. You are absolutely correct that they may have more functionality than an alias command.